.setup/CONFIGURE_SUBMITTY.py

```diff
@@ -132,7 +132,7 @@ def get_input(question, default=""):
 ##############################################################################
 
 defaults = {'database_host': 'localhost',
-            'database_user': 'hsdbu',
+            'database_user': 'submitty_dbuser',
```

Comment:
Yeah I agree, its alot of extra characters.  But actually since it is a user on the linux system, I would prefer to keep it specialized in name & function.  We have a special password just for this user.  Also, keeping the string unique allows us to grep for it in the code and its self-documenting.